src/CommunityToolkit.Maui/Views/Popup/PopupPage.shared.cs

```diff
@@ -176,7 +175,7 @@ void IQueryAttributable.ApplyQueryAttributes(IDictionary<string, object> query)
 
 	internal sealed partial class PopupPageLayout : Grid
 	{
-		public PopupPageLayout(in Popup popupContent, in IPopupOptions options)
+		public PopupPageLayout(in Popup popupContent, in IPopupOptions options, ICommand tapOutsideOfPopupCommand)
```

Comment:
Let's add the `in` keyword to align with the other parameters and ensure that `tapOutsideOfPopupCommand` is readonly.

```suggestion
		public PopupPageLayout(in Popup popupContent, in IPopupOptions options, in ICommand tapOutsideOfPopupCommand)
```